splunk_eventgen/lib/outputplugin.py

```diff
@@ -55,7 +55,10 @@ def run(self):
         if self.output_counter is not None:
             self.output_counter.collect(len(self.events), sum([len(e['_raw']) for e in self.events]))
         self.events = None
+        self._output_end()
 
+    def _output_end(self):
+        pass
```

Comment:
what is the purpose of this?

Reply:
this callback hook is added as a exit hook point for output plugin. 
This hook is used in the counter output plugin to collect information.
The counter plugin is used for debug and issue triaging scenario.